use/quickstart.md

```diff
@@ -5,28 +5,171 @@ permalink: /use/quickstart/
 redirect_from: /getting-started/
 ---
 
-# Apache Beam Quickstart
+# Apache Beam Java SDK Quickstart 
 
-The Apache Beam project is in the process of bootstrapping. This includes the creation of project resources, the refactoring of the initial code submission, and the formulation of project documentation, planning, and design documents. Until the project is fully initialized, this page contains useful resources to learn more about the model and tools which comprise Apache Beam.
+Run your first Beam pipeline to count words in a set of files. 
```

Comment:
meeting you both in the middle. these instructions rely on an archetype, so aren't totally general.